docs/components/style-spec/item.js

```diff
@@ -227,14 +227,30 @@ export default class Item extends React.Component {
                         </React.Fragment>
                     )}
 
-                    {this.props.function === 'interpolated' && (
+                    {this.props.expression && 
+                        (this.props.expression.interpolated || 
+                            this.props.expression.parameters.includes('feature-state')) && (
                         <React.Fragment>
                             Supports{' '}
-                            <a href="/mapbox-gl-js/style-spec/expressions/#interpolate">
+                            {this.props.expression.parameters.includes('feature-state') && (
+                                <em className="color-gray">
+                                    <a href="#expressions-feature-state">
```

Comment:
We'll want to update this path to point to the new expressions page:

```suggestion
                                    <a href="/mapbox-gl-js/style-spec/expressions/#feature-state">
```